src/field/pokemon.ts

```diff
@@ -4874,32 +4092,25 @@ export class PlayerPokemon extends Pokemon {
   addFriendship(friendship: number): void {
     if (friendship > 0) {
       const starterSpeciesId = this.species.getRootSpeciesId();
-      const fusionStarterSpeciesId = this.isFusion() && this.fusionSpecies ? this.fusionSpecies.getRootSpeciesId() : 0;
-      const starterData = [
-        globalScene.gameData.starterData[starterSpeciesId],
-        fusionStarterSpeciesId ? globalScene.gameData.starterData[fusionStarterSpeciesId] : null,
-      ].filter((d) => !!d);
+      const starterData = [globalScene.gameData.starterData[starterSpeciesId]].filter((d) => !!d);
```

Comment:
```suggestion
      const starterData = globalScene.gameData.starterData[starterSpeciesId];
```
I don't think this needs to be an array anymore